### DISCUSSION TITLE
fparam.npy not found

### DISCUSSION THREAD

**Discussion by WU2JING** [deepmodeling/deepmd-kit#441] [General]
When i try to train a model using dpgen, i meet a error  "fparam.npy not found".The detail can be find in the figure. 
![image](https://user-images.githubusercontent.com/68996784/112432778-5109f400-8d7c-11eb-9dd1-71b8875aab85.png)

**Comment by WU2JING** [comment]
The error fixed

**Comment by goldenhaaa** [comment]
Hello. May I ask how you prepare the fparam.npy file before training?
